library/opensuse

```diff
@@ -1,13 +1,9 @@
 # maintainer: Flavio Castelli <fcastelli@suse.com> (@flavio)
 
-# openSUSE 13.1
-13.1: git://github.com/openSUSE/docker-containers-build@0d21bc58cd26da2a0a59588affc506b977d6a846 docker
-bottle: git://github.com/openSUSE/docker-containers-build@0d21bc58cd26da2a0a59588affc506b977d6a846 docker
-
 # openSUSE 13.2
-13.2: git://github.com/openSUSE/docker-containers-build@cae38d2559dfb9698464bee2e9f4fa97ea017055 docker
-harlequin: git://github.com/openSUSE/docker-containers-build@cae38d2559dfb9698464bee2e9f4fa97ea017055 docker
-latest: git://github.com/openSUSE/docker-containers-build@cae38d2559dfb9698464bee2e9f4fa97ea017055 docker
+13.2: git://github.com/openSUSE/docker-containers-build@b3a02e025a99ecf124be4067e4ac94b2d5cf0ab3 docker
+harlequin: git://github.com/openSUSE/docker-containers-build@b3a02e025a99ecf124be4067e4ac94b2d5cf0ab3 docker
+latest: git://github.com/openSUSE/docker-containers-build@b3a02e025a99ecf124be4067e4ac94b2d5cf0ab3 docker
```

Comment:
Shouldn't these three be https://github.com/openSUSE/docker-containers-build/commit/25fa2adb969481d23f71e23ab7ff0148ae4df0ef instead? (`25fa2adb969481d23f71e23ab7ff0148ae4df0ef`)

The commit here is tumbleweed. :+1: